ChessCrafterDesktop/src/main/java/com/drewhannay/chesscrafter/utility/JavaFxFileDialog.java

```diff
@@ -44,12 +44,14 @@ public static File chooseFile(FileChooser.ExtensionFilter extensionFilter) {
 
     private static File doChooseDirectory() {
         DirectoryChooser directoryChooser = new DirectoryChooser();
+        directoryChooser.setInitialDirectory(new File(PreferenceUtility.getSaveLocationPreference()));
```

Comment:
This shouldn't be here. `JavaFxFileDialog` is a utility class that shouldn't depend on any of our other code. If we want a default location, we should pass it in as a parameter.